app/src/main/java/io/homeassistant/companion/android/widgets/media_player_controls/MediaPlayerControlsWidgetConfigureActivity.kt

```diff
@@ -202,9 +200,16 @@ class MediaPlayerControlsWidgetConfigureActivity : BaseActivity() {
 
             intent.putExtra(AppWidgetManager.EXTRA_APPWIDGET_ID, appWidgetId)
 
+            selectedEntities = LinkedList()
+            val se = binding.widgetTextConfigEntityId.text.split(",")
+            se.forEach {
+                val e = entities[it.trim()]
+                if (e != null) selectedEntities.add(e)
+            }
```

Comment:
I think a filter is better than a loop, but that's probably a personal preference.
```kotlin
selectedEntities = LinkedList(
    binding.widgetTextConfigEntityId.text.split(",")
        .mapNotNull { entities[it.trim()] }
)
```